oozie-to-airflow/mappers/shell_mapper.py

```diff
@@ -68,3 +68,7 @@ def convert_to_text(self) -> str:
     @staticmethod
     def required_imports() -> Set[str]:
         return {"from airflow.utils import dates", "from airflow.contrib.operators import dataproc_operator"}
+
```

Comment:
Can you extend the example that uses this mapper so that this property is also be tested integratively? Unfortunately, when there is no relations in the example, this property is not used.

Reply:
This proposal can be made in a separate PR.

Reply:
There is the `prepare` action in the example so there is a relation.
While I do agree there can be a separate PR for that, I've decided against it. That's because this change is an integral part of the whole PR where I prepare a working demo workflow.
I've split it in multiple separate commits, though, so that reviewing can be easier and when the PR is merged into the master branch, each change will be a separate and clear commit.